ckanext/csrf_filter/anti_csrf.py

```diff
@@ -95,6 +96,12 @@ def configure(config):
     token_expiry_age = 60 * config.get('ckanext.csrf_filter.token_expiry_minutes', 30)
     token_renewal_age = 60 * config.get('ckanext.csrf_filter.token_renewal_minutes', 10)
 
+    exempt_rules = [re.compile(r'^/+api/.*')]
+    custom_exempt_rules = config.get('ckanext.csrf_filter.exempt_rules', None)
+    if custom_exempt_rules:
+        for rule in json.loads(custom_exempt_rules):
```

Comment:
can you add a test for a invalid json load.

Reply:
Test for invalid json has been added